src/styles/main.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc((#{$day-size} * 7 + (6 * #{$gap})) + #{$padding} * 2);
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    cursor: pointer;
+    transition:
+      transform 0.25s,
+      background-color 0.25s;
+  }
+  &__day:hover {
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+  }
+
+  @for $num from 1 through 31 {
+    & :nth-child(#{$num})::before {
+      content: '#{$num}';
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      text-align: center;
+    }
```

Comment:
Inside the `@for` directive, it's better to use single quotes for string literals according to the code style rules. Also, ensure that the font-size does not hardcode pixels; it's recommended to use relative units like `rem` for scalability and accessibility. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)